diff --git a/server/gamemodes/vanilla/scripts/levelup.ts b/server/gamemodes/vanilla/scripts/levelup.ts
index c477c86..e15bf1d 100644
--- a/server/gamemodes/vanilla/scripts/levelup.ts
+++ b/server/gamemodes/vanilla/scripts/levelup.ts
@@ -152,10 +152,10 @@ function showSkillLevelUp(
                 uid: (LEVELUP_INTERFACE_ID << 16) | (targetComp & 0xffff),
                 archiveId: iconRef.archiveId,
                 frame: iconRef.frame,
-                x: 4,
-                y: 4,
-                width: 32,
-                height: 32,
+                x: 24,
+                y: 22,
+                width: 48,
+                height: 48,
             });
         }
     } catch (err) {
